### PR TITLE
Updated README for API call

### PR DESCRIPTION
**Pull request by seyilonge-nci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,181 @@ You must create you ticket branches off the NCIOCPL repo such that secrets are u
...
 
+## Making API Calls
+API endpoint domain has to be set as an initialization parameter for the application in `/src/index.js`.
+
+One API Domain Multiple Endpoints:
+--
+If a call needs to be made to an endpoint for the current weather data like the Open Weather Map API (https://api.openweathermap.org/data/2.5/weather?q=London).
+The "apiEndpoint" in `/src/index.js` can be set to "https://api.openweathermap.org/data/2.5".
+Once the "apiEndpoint" is set, the service name endpoint to get current weather data can be defined in `/src/services/api/endpoints.js` using the "getEndpoints" by adding one for the current weather data to "urls".
+```
```
Comment (arcepaul) [inline README.md:82]: why do we call it urls and not endpoints?

File: README.md [hunk 2]
```diff
@@ -71,6 +71,181 @@ You must create you ticket branches off the NCIOCPL repo such that secrets are u
...
+	return (
+		<>
+            {doneLoading && ({Content to be shown once fetch is complete})}
+            {!doneLoading && ({Show some visual loading state here while fetch is still in progress...})}
+        </>
+	);
+};
+
+export default Dashboard;
+```
```
Comment (arcepaul) [inline README.md:133]: If in the first example we show multiple endpoints being possible, I feel as though this blurb isn't necessary.

Comment (seyilonge-nci) [reply README.md:133]: With you on that. Will make the update.

File: README.md (hunk 1 above)
Comment (seyilonge-nci) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (seyilonge-nci) [reply README.md]: Updated

File: src/__tests__/App.test.js [hunk 3]
```diff
@@ -39,13 +39,15 @@ describe('App component', () => {
 		const basePath = '/';
 		const language = 'en';
 		const siteName = 'National Cancer Institute';
+		const weatherEndpoint = "https://api.openweathermap.org/data/2.5";
```
[the remark this answers is not in the document]
Comment (seyilonge-nci) [reply src/__tests__/App.test.js]: Arghhh! I knew that passing gravitational wave that appeared to distort the space time continuum messed something up during my commit. Fixed it :-)

File: src/App.js [hunk 4]
```diff
@@ -8,13 +8,9 @@ import './styles/app.scss';
...
 import { Home, ItemDetails, PageNotFound } from './views';
 
-
 const App = ({ tracking }) => {
 	// this should be a DUMB component that just displays our display(group) components
-	const {
-		HomePath,
-		ItemDetailsPath,
-	} = useAppPaths();
+	const { HomePath, ItemDetailsPath, WeatherPath } = useAppPaths();
```
[the remark this answers is not in the document]
Comment (seyilonge-nci) [reply src/App.js]: Fixed!

File: src/hooks/routing.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import { useStateValue } from '../store/store';
 const appPaths = {
 	HomePath: '/',
 	ItemDetailsPath: '/item/:id',
+	WeatherPath: '/weather',
```
[the remark this answers is not in the document]
Comment (seyilonge-nci) [reply src/hooks/routing.js]: Fixed!

**Comment by seyilonge-nci** [comment]
Added a table of contents.

**Pull request merged**